style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+.header {
+  margin-top: 10px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 15px 15px;
+}
+
+.brand {
+  width: 90px;
+  color: #28352f;
+  font-size: 20px;
+  text-align: center;
+  line-height: 28px;
+  height: 28px;
+  font-weight: 600;
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+  color: #28352f;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url(/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+}
+
+.headline-content {
+  padding-left: 24px;
+  padding-top: 64px;
+  width: 327px;
+  font-size: 40px;
+  text-align: center;
+  font-style: normal;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+  margin-bottom: 5px;
+}
+
+.intro-1 {
+  color: #172b4d;
+}
+
+.intro-2 {
+  color: #36b37e;
+}
+
+.description {
+  width: 327px;
+  height: 164px;
+  padding-left: 24px;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  font-style: normal;
+  color: #172b4d;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  line-height: 28px;
+}
```

Comment:
- [ ] Kindly make sure to add the font family for description according to the Figma design.